modules/git/signature_gogit.go

```diff
@@ -21,38 +21,59 @@ type Signature = object.Signature
 // Helper to get a signature from the commit line, which looks like these:
 //
 //	author Patrick Gundlach <gundlach@speedata.de> 1378823654 +0200
-//	author Patrick Gundlach <gundlach@speedata.de> Thu, 07 Apr 2005 22:13:13 +0200
+//	author Patrick Gundlach <gundlach@speedata.de> Thu Apr 07 22:13:13 2005 +0200
```

Comment:
Why the code is different from nogogit? And it seems that the nogogit function does pretty well on parsing the date/time:


![image](https://github.com/go-gitea/gitea/assets/2114189/b6b1b621-35b3-4477-b0d4-74eed8e9c845)